src/App.tsx

```diff
@@ -1,61 +1,110 @@
+import { useState } from 'react';
 import './App.scss';
+import { TodoList } from './components/TodoList';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
 
 export const App = () => {
+  const [todos, setTodos] = useState([...todosFromServer]);
+  const [title, setTitle] = useState('');
+  const [user, setUser] = useState('');
+  const [users] = useState([...usersFromServer]);
+  const [isTitleError, setIsTitleError] = useState(false);
+  const [isUserError, setIsUserError] = useState(false);
+
+  const reset = () => {
+    setTitle('');
+    setUser('');
+    setIsTitleError(false);
+    setIsUserError(false);
+  };
+
+  const handleSubmit = (e: React.FormEvent<HTMLFormElement>) => {
+    e.preventDefault();
+
+    if (title.trim() && user) {
+      const todo = {
+        id: Math.max(...todos.map((t) => t.id)) + 1,
+        title,
+        userId: +user,
+        completed: false,
+      };
+
+      setTodos([...todos, todo]);
+      reset();
+    } else {
+      if (!title.trim()) {
+        setIsTitleError(true);
+      }
+
+      if (!user) {
+        setIsUserError(true);
+      }
+    }
+  };
+
+  const handleTitle = (e: React.ChangeEvent<HTMLInputElement>) => {
```

Comment:
```suggestion
  const handleChangeTitle = (e: React.ChangeEvent<HTMLInputElement>) => {
```
the same in the function below, just FYI